### ISSUE TITLE
ONN doesn't seem to converge

### ISSUE THREAD

**Issue by skaftanis** [alison-carrera/onn#8]
I'm trying the simple Online Neural Network (ONN). I'm try to do multiple partial fit with your test data in your README, but it's not converging. 
For example when I run 100 times the partial_fit function with [0.1,0.2] -> 0 and [0.8,0.5] -> 1 I always have zero (0) as the predictions, so the training accuracy is 50% in this toy example for some reason. 

Am I doing something wrong here? 
Thanks!

**Comment by alison-carrera** [comment]
Hmm, any news? Sorry for the delay, I was very busy these days.

I will have a look at this. :D

**Comment by alison-carrera** [comment]
@skaftanis The example in the README is just a fake example of how you need to input your data into the function. It will not converge with just these data. You can test with generated data from scikit-learn for test purposes. Have a look at the jupyter notebook example in the code, there will be everything you need to know about how to use the algorithm. 

[ONN_Usage_Example](https://github.com/alison-carrera/onn/blob/master/ONN_Usage_Example.ipynb)

Thanks!

**Comment by alison-carrera** [comment]
Closing this issue because no answers were given.
